<ui version="4.0" >
 <class>PersonalInviteWidget</class>
 <widget class="QWidget" name="PersonalInviteWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" colspan="3" >
    <widget class="K3ActiveLabel" native="1" name="mainTextLabel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>34</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="6" row="0" column="0" >
    <widget class="QLabel" name="pixmapLabel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>108</width>
       <height>318</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>108</width>
       <height>318</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" >
    <widget class="K3ActiveLabel" native="1" name="hostLabel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLabel" name="kActiveLabel6" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&lt;b>Password:&lt;/b></string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLabel" name="kActiveLabel7" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&lt;b>Expiration time:&lt;/b></string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="K3ActiveLabel" native="1" name="passwordLabel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="K3ActiveLabel" native="1" name="expirationLabel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="kActiveLabel5" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&lt;b>Host:&lt;/b></string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="K3ActiveLabel" native="1" name="hostHelpLabel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>K3ActiveLabel</class>
   <extends>QWidget</extends>
   <header>k3activelabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
